catalog/samples/basic_app_bar.md

```diff
@@ -58,15 +59,22 @@ class _BasicAppBarSampleState extends State<BasicAppBarSample> {
         appBar: new AppBar(
           title: const Text('Basic AppBar'),
           actions: <Widget>[
-            new IconButton( // action button
+            new IconButton(
+              // action button
```

Comment:
nit here and elsewhere: should comments like this maybe go before the element they are referring to?